app/src/main/java/ru/nke/collegeschedule/schedule/ui/authentication/AuthenticationFragment.kt

```diff
@@ -0,0 +1,196 @@
+package ru.nke.collegeschedule.schedule.ui.authentication
+
+import android.Manifest
+import android.content.pm.PackageManager
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Toast
+import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.core.content.ContextCompat
+import androidx.fragment.app.Fragment
+import androidx.navigation.Navigation
+import ru.nke.collegeschedule.schedule.R
+import ru.nke.collegeschedule.schedule.databinding.AuthenticationFragmentBinding
+import ru.nke.collegeschedule.schedule.util.autoCleared
+
+const val cameraRequestCode = 1888
+const val authStringScanResult = "scan_result"
+const val authStringScanTask = "task_to_scan"
+const val authStringError = "error_string"
```

Comment:
```suggestion
const val CAMERA_REQUEST_CODE = 1888
const val AUTH_SCAN_RESULT_CODE = "scan_result"
const val AUTH_SCAN_TASK_CODE = "task_to_scan"
const val AUTH_ERROR_CODE = "error_string"
```

Также, не забудь исправить использование эти констант в коде.